case-uco/java/src/org/sleuthkit/caseuco/CaseUcoExporter.java

```diff
@@ -590,13 +590,13 @@ private void assembleGenInfo(String uuid, BlackboardArtifact artifact, List<Json
     }
 
     private void assembleWebHistory(String uuid, BlackboardArtifact artifact, List<JsonElement> output) throws TskCoreException {
-        Trace userNameNode = new BlankTraceNode();
+        ObservableObject userNameNode = new BlankTraceNode();
```

Comment:
@markmckinnon We should probably rename BlankTraceNode to BlankObservableObject.

Reply:
Changed and commited.